targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_IMX/CMakeLists.txt

```diff
@@ -10,20 +10,14 @@ target_include_directories(mbed-imx
 
 target_sources(mbed-imx
     INTERFACE
-        analogin_api.c
```

Comment:
why are these being moved from IMX to MIMXRT ?

Reply:
The drivers that are called from the SDK are different and have different name for these two different MIMXRT devices.

Reply:
Also some of the macros and variables differ from these two devices, so they need separate apis.

Reply:
@0xc0170  Is it ok to start the CI ?